Magento2/Tests/Methods/DeprecatedModelMethodUnitTest.inc

```diff
@@ -0,0 +1,4 @@
+<?php
+$model->getResource()->save();
```

Comment:
Please add positive cases as well.

Reply:
Thanks @lenaorobei 
Can you please suggest a negative case that I can add?

Reply:
@milindsingh,
What about this?
```php
$model->getResource()->myCustomMethod();
$model->myCustomMethod();
```